src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2315,6 +2377,97 @@
             "required": [ "name" ]
         },
 
+        "toolComponentTranslation": {
+            "description": "TBD.",
+            "additionalProperties": false,
+            "type": "object",
+            "properties": {
+
+                "toolComponentGuid": {
+                    "description": "TBD must be GUID format: matches toolComponent.guid.",
+                    "type": "string"
+                },
+
+                "location": {
+                    "description": "The location where translation can be obtained.",
+                    "$ref": "#/definitions/artifactLocation"
+                },
+
+                "semanticVersion": {
+                    "description": "The semantic version of the tool component for which the translation was made.",
+                    "type": "string"
+                },
+
+                "partialTranslation": {
+                    "description": "Should be true if this object contains a subset of the strings defined by the tool component.",
+                    "type": "boolean"
+                },
+
+                "globalMessageStrings": {
+                    "description": "A set of name/value pairs with property names as a subset of the names in toolComponent.globalMessageStrings.",
```

Comment:
>description [](start = 21, length = 11)

Copy the description of `toolComponent.globalMessageStrings` (changing "resource identifier" to "message identifier" as I commented above). Then add a sentence: "The property names are a subset of the property names in the globalMessageStrings property of the toolComponent object to which this translation belongs."